### PR TITLE
Laravel 8

### PR DESCRIPTION
**Pull request by matthewnessworthy**

### CODE REVIEW

File: src/ExperimentsServiceProvider.php [hunk 1]
```diff
@@ -8,15 +8,27 @@ class ExperimentsServiceProvider extends ServiceProvider
...
+        }
     }
 
     public function register()
     {
-        $this->commands(
-            [
-                Console\InstallCommand::class,
-            ]
+        $this->mergeConfigFrom(
```
Comment (lucidlogic) [inline src/ExperimentsServiceProvider.php:29]: please don't add mergeConfigFrom

**Review by lucidlogic — CHANGES_REQUESTED** [review]
please don't add mergeConfigFrom

File: src/ExperimentsServiceProvider.php (hunk 1 above)
Comment (matthewnessworthy) [reply src/ExperimentsServiceProvider.php:29]: why? (just asking so i know for any future PRs)

Comment (lucidlogic) [reply src/ExperimentsServiceProvider.php:29]: there are some boilerplate example experiments in there. and these will actually be run if merged from the package.
```
'your-experiment-key' => [
        'control' => 'control-key',
        'test' => 'test-key',
    ],
```

Comment (matthewnessworthy) [reply src/ExperimentsServiceProvider.php:29]: this does mean that new configs added to this package will be ignored by the downstream package if they have dumped the config

and i can just comment out the example experiment

Comment (lucidlogic) [reply src/ExperimentsServiceProvider.php:29]: no leave please

Comment (lucidlogic) [reply src/ExperimentsServiceProvider.php:29]: if I need a config added I will add mergeConfigFrom

Comment (matthewnessworthy) [reply src/ExperimentsServiceProvider.php]: done

File: src/ExperimentsServiceProvider.php [hunk 2]
```diff
@@ -8,16 +8,28 @@ class ExperimentsServiceProvider extends ServiceProvider
...
+     * {@inheritdoc}
+     */
     public function register()
     {
-        $this->commands(
-            [
-                Console\InstallCommand::class,
-            ]
-        );
+        //
```
Comment (lucidlogic) [inline src/ExperimentsServiceProvider.php:32]:
```suggestion
```

**Review by lucidlogic — APPROVED** [review]

**Pull request merged**